x-pack/plugin/ent-search/qa/rest/roles.yml

```diff
@@ -28,14 +28,15 @@ admin:
         "test-nonexistent-search-application",
         "test-search-application-with-invalid-template"
     ]
-      privileges: [ "manage", "write" ]
+      privileges: [ "manage", "write", "read" ]
```

Comment:
We'd have to be very granular and create new users with limited role privileges. Since this is an `admin` role anyway I think a separate role to add read privileges would be overkill here.

Reply:
My concern is that we probably have used these indices too extensively in other tests, and now we're unclear on what privileges are needed on the different tests.

I would prefer that we keep the privileges to a minimum as much as possible, so in case it's easy to add the new privilege to the specific indices, I would feel better. Maybe we need to start segregating the indices we're using for each API (search apps, rules, analytics) 🤷

Reply:
I have `test-index1` with read privileges in the user role already, and I ensured that I have tests that ensure a user can acces them. Are you suggesting that all tests be using the user role and no tests use the admin role? Can you please help me understand the benefits to that here?

Reply:
Resolving this in favor of a team [discussion](https://github.com/elastic/enterprise-search-team/discussions/5146).

Reply:
Thanks for opening the discussion!

I think my confusion comes from changing 19 indices privileges for the `admin` role. As you mentioned, this is an `admin` role, so that might not be a concern as it is expected to have permissions on all indices.

Should we change the admin role to have permissions on all indices then? Something like

```yaml
admin:
  cluster:
    - manage_search_application
    - manage_behavioral_analytics
    - manage
    - monitor
  indices:
    - names: [
        # indices
        "test-*"
    ]
      privileges: [ "manage", "write", "read"]
```

Reply:
I like that suggestion, I'll do that for now and we can revisit later if we decide to!